index.html

```diff
@@ -1535,7 +1535,7 @@ <h1>Event Partners</h1>
 											<a href="https://github.com/fossasia"><i class="icon fa fa-github"></i></a>
 											<a href="https://www.facebook.com/fossasia/"><i class="icon social_facebook"></i></a>
 											<a href="https://in.linkedin.com/company/fossasia"><i class="icon social_linkedin"></i></a>
-											<!-- <a href="#"><i class="icon social_linkedin"></i></a> -->
+											<a href="https://in.linkedin.com/company/fossasia"><i class="icon social_linkedin"></i></a>
```

Comment:
@0x48piraj  Why there are two similar accounts?

Reply:
One is for the top and one is for below.